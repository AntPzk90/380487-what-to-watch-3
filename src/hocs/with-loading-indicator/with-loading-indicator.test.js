import React from 'react';
import renderer from 'react-test-renderer';
import withLoadingIndicatior from './with-loading-indicator.jsx';


it(`SnapshotTest withloadingIndicator`, () => {

  const tree = renderer
  .create(
      <withLoadingIndicatior/>
  )
  .toJSON();

  expect(tree).toMatchSnapshot();
});
