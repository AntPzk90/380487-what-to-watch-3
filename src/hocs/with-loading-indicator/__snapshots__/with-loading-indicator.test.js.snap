// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotTest withloadingIndicator 1`] = `<withLoadingIndicatior />`;
